.github/workflows/benchmark.yml

```diff
@@ -8,15 +8,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        config_dataset_info: [
-            "sift_front_end,  door-12,               12,  JPG,  test_data,  olsson-loader,  1296",
```

Comment:
let's keep the old ones, since the compute is free, and doesn't constrain the length of the CI, and nice to keep around for future regression testing.

Reply:
Done